### PR TITLE
KBC-2751 tables definition endpoint identifies typed table

### PR DESCRIPTION
**Pull request by martinjunger**
Jira: KBC-2751

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [ ] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Backend/Exasol/TableDefinitionOperationsTest.php [hunk 1]
```diff
@@ -285,6 +285,281 @@ public function testDataPreviewForTableDefinitionWithDecimalType(): void
...
+            'column_float',
+            'column_boolean',
+            'column_date',
+            'column_timestamp',
+            'column_varchar',
+        ]);
+        $csvFile->writeRow(
+            [
+                '1',
+                '0.123', // default is (36,36) => 1e-36
```
Comment (martinjunger) [inline tests/Backend/Exasol/TableDefinitionOperationsTest.php:340]: BTW defaultni velikost pro NUMERIC je "36,36", coz je "0.000000000...0" (0 cislic,36 cislic).
Default je "18,0", coz je "18 cislic,0 cislic".
Lepsi by bylo mit napr. "18,18", teda "18cislic,18cislic" - napr. pro castky dostacujici.

Comment (jirkasemmler) [reply tests/Backend/Exasol/TableDefinitionOperationsTest.php:342]: ani jedno neni reseni, default by melo byt 36,18. Tzn celkem 36 cifer, ale 18 pred, 18 za desetinnou teckou

Comment (jirkasemmler) [reply tests/Backend/Exasol/TableDefinitionOperationsTest.php:342]: https://keboola.atlassian.net/browse/KBC-2782

File: tests/Backend/Exasol/TableDefinitionOperationsTest.php [hunk 2]
```diff
@@ -285,6 +285,281 @@ public function testDataPreviewForTableDefinitionWithDecimalType(): void
...
+        $expectedPreview = [
+            [
+                [
+                    'columnName' => 'id',
+                    'value' => '1',
+                    'isTruncated' => false,
+                ],
+                [
+                    'columnName' => 'column_decimal',
+                    // TODO actual: 0.122999999999999995035135668316536832
```
[the remark this answers is not in the document]
Comment (jirkasemmler) [reply tests/Backend/Exasol/TableDefinitionOperationsTest.php:367]: imho na to reseni uplne neni

**Review by jirkasemmler — APPROVED** [review]
lgtm

**Pull request merged**